utils/frame/benchmarking-cli/src/pallet/command.rs

```diff
@@ -456,51 +460,51 @@ impl PalletCmd {
 			if !self.no_storage_info {
 				let mut comments: Vec<String> = Default::default();
 				writer::add_storage_comments(&mut comments, &batch.db_results, storage_info);
-				println!("Raw Storage Info\n========");
+				log::warn!(target:LOG_TARGET,"Raw Storage Info\n========");
 				for comment in comments {
-					println!("{}", comment);
+					log::warn!(target:LOG_TARGET,"{}", comment);
 				}
-				println!();
```

Comment:
i think these should also be println, since it is part of a command flag to output this information